intern/service/boss/v1/volume.py

```diff
@@ -96,20 +97,38 @@ def create_cutout(
                 block_size=(1024, 1024, 32)
             )
 
-            for b in blocks:
-                _data = np.ascontiguousarray(
-                    numpyVolume[
-                        b[2][0] - z_range[0]: b[2][1] - z_range[0],
-                        b[1][0] - y_range[0]: b[1][1] - y_range[0],
-                        b[0][0] - x_range[0]: b[0][1] - x_range[0]
-                    ],
-                    dtype=numpyVolume.dtype
-                )
-                self.create_cutout(
-                    resource, resolution, b[0], b[1], b[2],
-                    time_range, _data, url_prefix, auth, session, send_opts
-                )
-            return
+            if parallel:
+                pool = multiprocessing.Pool(processes=parallel if isinstance(parallel, int) and parallel > 0 else multiprocessing.cpu_count())
+                pool.starmap(self.create_cutout, [
+                    (
+                        resource, resolution, b[0], b[1], b[2],
+                        time_range, 
+                        np.ascontiguousarray(
+                            numpyVolume[
+                                b[2][0] - z_range[0]: b[2][1] - z_range[0],
+                                b[1][0] - y_range[0]: b[1][1] - y_range[0],
+                                b[0][0] - x_range[0]: b[0][1] - x_range[0]
+                            ],
+                            dtype=numpyVolume.dtype
+                        ), 
+                        url_prefix, auth, session, send_opts,
+                    )
+                 for b in blocks])
+            else:
+                for b in blocks:
+                    _data = np.ascontiguousarray(
```

Comment:
Can we make a function for creating the Numpy array so we can use it here and above?

Reply:
Most definitely. To include:

* Making contiguous with correct dtype
* With specific sub-index

Anything else?

Reply:
Basically just the params needed to make the numpy array.

Reply:
I think this can be marked resolved now :)